src/legacy/core_plugins/kibana/public/dashboard/np_ready/dashboard_app_controller.tsx

```diff
@@ -270,6 +271,7 @@ export class DashboardAppController {
         viewMode: dashboardStateManager.getViewMode(),
         panels: embeddablesMap,
         isFullScreenMode: dashboardStateManager.getFullScreenMode(),
+        isEmbedded: Boolean($routeParams.embed),
```

Comment:
@majagrubic would it be correct to call `isEmbedded` :arrow_right: `isShared` \ `isExternal` \ `isEmbeddedExternally`?

Reply:
`isEmbeddedExternally` sounds good.